modules/aws/awc/main.tf

```diff
@@ -10,6 +10,7 @@ locals {
 
   provisioning_script = "awc-provisioning.sh"
   awm_script          = "get-connector-token.py"
+  log_manager_script  = "connector_log_manager.py"
```

Comment:
The usage of `var.local_manager_script` and `local.local_manager_script` is mixed in the code.

A `local` is used within the `aws_s3_object.log-manager-script` resource on line [78](https://github.com/EditShareCo/cloud_deployment_scripts_hp_inc/pull/5/files#diff-51d281d110c60958957a4a5e352dd6e2aa0aa02de13680b3df9564a0dfbe6852R75) and the `local` used again in the resources list of it on line [172](https://github.com/EditShareCo/cloud_deployment_scripts_hp_inc/pull/5/files#diff-51d281d110c60958957a4a5e352dd6e2aa0aa02de13680b3df9564a0dfbe6852R172)
but the `variable` version of the same string is used in the `aws_s3_object.awc-provisioning-script` resource on line [121](https://github.com/EditShareCo/cloud_deployment_scripts_hp_inc/pull/5/files#diff-51d281d110c60958957a4a5e352dd6e2aa0aa02de13680b3df9564a0dfbe6852R121)

Is there a reason/purpose I have missed why we use a mixed economy and/or don't pass the module var through to the local so its used everywhere?

Reply:
changed to use var.log_manager_script throughout